projects/aws/image-builder/builder/builder.go

```diff
@@ -155,6 +155,12 @@ func (b *BuildOptions) BuildImage() {
 
 		outputArtifactPath = filepath.Join(cwd, fmt.Sprintf("%s.gz", b.Os))
 	} else if b.Hypervisor == Nutanix {
+		// Patch firmware config for tool
+		upstreamPatchCommand := fmt.Sprintf("make -C %s image-builder/eks-anywhere-patched", imageBuilderProjectPath)
```

Comment:
```suggestion
		upstreamPatchCommand := fmt.Sprintf("make -C %s patch-repo", imageBuilderProjectPath)
```

Reply:
thank @jaxesn . wondering if the same update is required on line 109 as well for bare-metal case? cc @vignesh-goutham

Reply:
Yea sure, you can make that change as well. Thanks.